polytope/polytope.py

```diff
@@ -609,43 +608,43 @@ def solve_rotation_ap(u, v):
     # product is non-zero
 
     N = u.size  # the number of dimensions
-    uv = np.vstack([u, v])  # the plane of rotation
+    uv = np.stack([u, v], axis=1)  # the plane of rotation
```

Comment:
The function [`numpy.stack` was introduced in `numpy == 1.10.0`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.stack.html). However `install_requires` was at `numpy >= 1.6` in . The tests fail with errors of the form:

```python
======================================================================
6) ERROR: polytope_test.solve_rotation_test_270L
----------------------------------------------------------------------
   Traceback (most recent call last):
    /some/path/.virtualenvs/dev/lib/python3.5/site-packages/nose/case.py line 198 in runTest
      self.test(*self.arg)
    polytope_test.py line 283 in solve_rotation_test_270L
      R = solve_rotation_ap(g1, g2)
    /some/other/path/polytope/polytope/polytope.py line 579 in solve_rotation_ap
      uv = np.stack([u, v], axis=1)  # the plane of rotation
   AttributeError: module 'numpy' has no attribute 'stack'
```

A [later bump](https://github.com/tulip-control/polytope/commit/daddd18a15c2c9cbe04f4b0ffd0204f127a84229) to require `numpy >= 1.7.1` was based on what `scipy` required.